charts/neon-storage-controller/templates/deployment.yaml

```diff
@@ -57,6 +57,9 @@ spec:
           {{- if .Values.settings.computeHookUrl }}
             - --compute-hook-url
             - {{ .Values.settings.computeHookUrl | quote }}
+          {{- else if .Values.settings.controlPlaneUrl }}
+            - --compute-hook-url
```

Comment:
please pass it through verbatim via `--control-plane-url`, instead of using `--compute-hook-url`